solidity/test/TBTCSystemTest.js

```diff
@@ -9,6 +9,10 @@ const TBTCSystem = contract.fromArtifact("TBTCSystem")
 const SatWeiPriceFeed = contract.fromArtifact("SatWeiPriceFeed")
 const MockMedianizer = contract.fromArtifact("MockMedianizer")
 
+function btcToBTC18(n) {
```

Comment:
This is really btcToTbtc, no?

Reply:
If that's the convention we're following in the codebase, yeah. I wish we had an explicit name for 10^-18 BTC unit though...

Reply:
Addressed.